spec.v1.yml

```diff
@@ -0,0 +1,364 @@
+openapi: 3.0.0
+info:
+  version: "1.0"
+  title: Sunnah.com API
+  description: >-
+    Sunnah.com API offers access to verified data of collections of hadith of
+    the Prophet (SAWS). Here you will find a list of the available endpoints in
+    our REST API.
+
+
```

Comment:
remove extra empty lines?

Reply:
Looks like we need it to have the line break appear in swagger UI, if removing the newline it disappears.